gradle/versions.gradle

```diff
@@ -12,47 +12,46 @@ dependencyResolutionManagement {
             version('springCloudCB', '3.1.0')
             version('springCloudGateway', '4.1.1')
-            version('springFramework', '6.1.3')
-            version('springRetry', '1.3.4')
+            version('springFramework', '6.1.5')
+            version('springRetry', '2.0.5')
             version('springWebTestClient', '5.4.0')
 
             version('zosUtils', '2.0.4')
             version('archaius', '0.7.9')
             version('awaitility', '4.2.0')
             version('bouncyCastle', '1.77')
-            version('bootstrap', '4.3.1')
-            // forced version in root gradle.build file. Version 3.x requieres Java 11
-            version('caffeine', '2.9.3')
+            version('caffeine', '3.1.8')
             version('commonsLang3', '3.14.0')
             version('commonsText', '1.11.0')
             version('commonsIo', '2.15.1')
             version('ehCache', '3.10.8')
-            version('eureka', '2.0.1')
+            version('eureka', '2.0.2')
             version('gradleGitProperties', '2.4.1') // Used in classpath dependencies
             version('gradleNode', '3.6.0') // Used in classpath dependencies
             version('guava', '33.0.0-jre')
             version('hamcrest', '1.3')
             version('httpClient5', '5.3')
-            version('infinispan', '14.0.22.Final')
-            version('jacksonCore', '2.16.1')
-            version('jacksonDatabind', '2.16.1')
-            version('jacksonDataformatYaml', '2.16.1')
+            version('infinispan', '14.0.25.Final')
+            version('jacksonCore', '2.17.0')
+            version('jacksonDatabind', '2.17.0')
+            version('jacksonDataformatYaml', '2.17.0')
             version('janino', '3.1.11')
-            version('jakartaValidation', '3.0.2')
-            version('jakartaInject', '1.0.5')
-            version('jakartaServlet', '6.1.0-M1')
-            version('jaxbApi', '2.3.1')
+            version('jakartaValidation', '3.1.0-M1')
+            version('jakartaInject', '2.0.1')
+            version('jakartaServlet', '6.1.0-M2')
+            version('jaxbApi', '2.4.0-b180830.0359')
             version('jerseySun', '1.19.4')
-            version('jettyWebSocketClient', '9.4.53.v20231009')
+            version('jettyWebSocketClient', '9.4.54.v20240208')
             version('jettison', '1.5.4')
-            version('jjwt', '0.11.5')
+            //0.12.x version contains breaking changes
+            version('jjwt', '0.12.5')
             version('jsonPath', '2.8.0')
             version('jsonSmart', '2.5.0')
             version('junitJupiter', '5.10.1')
-            version('junitPlatform', '1.10.1')
-            version('lettuce', '6.3.0.RELEASE')
+            version('junitPlatform', '1.10.2')
+            version('lettuce', '6.3.1.RELEASE')
             // force version in build.gradle file - compatibility with Slf4j
-            version('log4j', '2.22.0')
+            version('log4j', '3.0.0-beta2')
```

Comment:
this version seems to have vulnerabilities and it is a beta, it would be better to go with 2.23.1